test/unit/dfc_test.py

```diff
@@ -193,7 +201,7 @@ def named_empty_df(df_name):
             names = ["a", "b", "c", "d"]
             res = dfc.merge_list(named_empty_df.map(names))
         final = f.run().result[res].result
-        self.assertEqual(["a", "b", "c", "d"], final.get_table_names())
+        assert final.get_table_names() == ["a", "b", "c", "d"]
 
     def test_fanout_task(self):
```

Comment:
I'm bewildered by this test. I guess I need to go look at the DFC class. Do you understand what fanout is doing, and the work that Prefect is doing in here?

Reply:
Fanout task simply splits DFC into list of DFC with fixed number of tables each.

Reply:
This could be used in situations like epacems where large number of tables come together in a single dfc and where you want to process this in parallel by splitting it into smaller chunks that can be processed in parallel.